src/main/java/Main.java

```diff
@@ -1,19 +1,13 @@
-import java.io.File;
-import java.io.InputStream;
-
 /*
```

Comment:
I didn't work on this plugin before. Is there a good reason to keep this file? Otherwise feel free to remove it.